keyboards/planck/info.json

```diff
@@ -5,107 +5,107 @@
-        { "w": 1, "x": 4, "y": 1 }, 
-        { "w": 1, "x": 5, "y": 1 }, 
-        { "w": 1, "x": 6, "y": 1 }, 
-        { "w": 1, "x": 7, "y": 1 }, 
-        { "w": 1, "x": 8, "y": 1 }, 
-        { "w": 1, "x": 9, "y": 1 }, 
-        { "w": 1, "x": 10, "y": 1 }, 
-        { "w": 1, "x": 11, "y": 1 }, 
-        { "w": 1, "x": 0, "y": 2 }, 
-        { "w": 1, "x": 1, "y": 2 }, 
-        { "w": 1, "x": 2, "y": 2 }, 
-        { "w": 1, "x": 3, "y": 2 }, 
-        { "w": 1, "x": 4, "y": 2 }, 
-        { "w": 1, "x": 5, "y": 2 }, 
-        { "w": 1, "x": 6, "y": 2 }, 
-        { "w": 1, "x": 7, "y": 2 }, 
-        { "w": 1, "x": 8, "y": 2 }, 
-        { "w": 1, "x": 9, "y": 2 }, 
-        { "w": 1, "x": 10, "y": 2 }, 
-        { "w": 1, "x": 11, "y": 2 }, 
-        { "w": 1, "x": 0, "y": 3 }, 
-        { "w": 1, "x": 1, "y": 3 }, 
-        { "w": 1, "x": 2, "y": 3 }, 
-        { "w": 1, "x": 3, "y": 3 }, 
-        { "w": 1, "x": 4, "y": 3 }, 
-        { "w": 2, "x": 5, "y": 3 }, 
-        { "w": 1, "x": 7, "y": 3 }, 
-        { "w": 1, "x": 8, "y": 3 }, 
-        { "w": 1, "x": 9, "y": 3 }, 
-        { "w": 1, "x": 10, "y": 3 }, 
+      "layout": [
+        { "w": 1, "x": 0, "y": 0 },
+        { "w": 1, "x": 1, "y": 0 },
+        { "w": 1, "x": 2, "y": 0 },
+        { "w": 1, "x": 3, "y": 0 },
+        { "w": 1, "x": 4, "y": 0 },
+        { "w": 1, "x": 5, "y": 0 },
+        { "w": 1, "x": 6, "y": 0 },
+        { "w": 1, "x": 7, "y": 0 },
+        { "w": 1, "x": 8, "y": 0 },
+        { "w": 1, "x": 9, "y": 0 },
+        { "w": 1, "x": 10, "y": 0 },
+        { "w": 1, "x": 11, "y": 0 },
+        { "w": 1, "x": 0, "y": 1 },
+        { "w": 1, "x": 1, "y": 1 },
+        { "w": 1, "x": 2, "y": 1 },
+        { "w": 1, "x": 3, "y": 1 },
+        { "w": 1, "x": 4, "y": 1 },
+        { "w": 1, "x": 5, "y": 1 },
+        { "w": 1, "x": 6, "y": 1 },
+        { "w": 1, "x": 7, "y": 1 },
+        { "w": 1, "x": 8, "y": 1 },
+        { "w": 1, "x": 9, "y": 1 },
+        { "w": 1, "x": 10, "y": 1 },
+        { "w": 1, "x": 11, "y": 1 },
+        { "w": 1, "x": 0, "y": 2 },
+        { "w": 1, "x": 1, "y": 2 },
+        { "w": 1, "x": 2, "y": 2 },
+        { "w": 1, "x": 3, "y": 2 },
+        { "w": 1, "x": 4, "y": 2 },
+        { "w": 1, "x": 5, "y": 2 },
+        { "w": 1, "x": 6, "y": 2 },
+        { "w": 1, "x": 7, "y": 2 },
+        { "w": 1, "x": 8, "y": 2 },
+        { "w": 1, "x": 9, "y": 2 },
+        { "w": 1, "x": 10, "y": 2 },
+        { "w": 1, "x": 11, "y": 2 },
+        { "w": 1, "x": 0, "y": 3 },
+        { "w": 1, "x": 1, "y": 3 },
+        { "w": 1, "x": 2, "y": 3 },
+        { "w": 1, "x": 3, "y": 3 },
+        { "w": 1, "x": 4, "y": 3 },
+        { "w": 2, "x": 5, "y": 3 },
+        { "w": 1, "x": 7, "y": 3 },
+        { "w": 1, "x": 8, "y": 3 },
+        { "w": 1, "x": 9, "y": 3 },
+        { "w": 1, "x": 10, "y": 3 },
         { "w": 1, "x": 11, "y": 3 } ]
     },
-    "LAYOUT_planck_grid": {
+    "LAYOUT_ortho_4x12": {
       "key_count": 48,
```

Comment:
```suggestion
```
Just while we're here :)

Reply:
Ah, didn't know that that property was optional (as per `docs/reference_info_json.md`)